src/Tracker/index.js

```diff
@@ -97,8 +96,44 @@ const formatProperties = properties => {
   )
 }
 
-const userAnalyticsEvent = (eventName, properties) =>
+const userAnalyticsEvent = (eventName, properties) => {
+  switch (eventName) {
+    case ('screen_welcome'):
+      eventName = 'WELCOME'
+      break
+    case ('screen_document_front_capture_file_upload'):
+      eventName = 'DOCUMENT_CAPTURE_FRONT'
+      break
+    case ('screen_document_front_confirmation'):
+      eventName = 'DOCUMENT_CAPTURE_CONFIRMATION_FRONT'
+      break
+    case ('screen_document_back_capture_file_upload'):
+      eventName = 'DOCUMENT_CAPTURE_BACK'
+      break
+    case ('screen_document_back_confirmation'):
+      eventName = 'DOCUMENT_CAPTURE_CONFIRMATION_BACK'
+      break
+    case ('screen_face_selfie_intro'):
+      eventName = 'FACIAL_INTRO'
+      break
+    case ('screen_face_selfie_confirmation'):
+      eventName = 'FACIAL_CAPTURE_CONFIRMATION'
+      break
+    case ('screen_face_video_intro'):
+      eventName = 'VIDEO_FACIAL_INTRO'
+      break
+    case ('screen_face_video_capture_step_1'):
+      eventName = 'VIDEO_FACIAL_CAPTURE_STEP_1'
+      break
+    case ('screen_face_video_capture_step_2'):
+      eventName = 'VIDEO_FACIAL_CAPTURE_STEP_2'
+      break
+    case ('Starting upload'):
+      eventName = 'UPLOAD'
+      break
+  }
```

Comment:
With this approach the `eventName` parameter is being modified. Better to return it as a new variable. Also would be more compact code wise as a map of key values to look up.

Reply:
Updated